### DISCUSSION TITLE
Bridge Reopening

### DISCUSSION THREAD

**Discussion by jettblu** [aurora-is-near/rainbow-bridge#803] [General] [answered]
Hello! The Rainbow Bridge UI indicates the bridge is currently closed due to the merge. Does anyone have a tentative timeline for the bridge reopening?

**Comment by sept-en** [comment, marked as answer]
@jettblu only interaction with the Ethereum network is paused. As we announced in our weekly updates, the Ethereum network on Rainbow Bridge will be available next week.

**Reply by unrelated19** [reply]
any update on this ser?

**Reply by sept-en** [reply]
@nedsa548 now it's live: https://twitter.com/auroraisnear/status/1574573522453639168
